packages/glimmer-runtime/tests/updating-test.ts

```diff
@@ -547,6 +547,28 @@ test("class attribute w/ concat follow the normal dirtying rules", function() {
   equalTokens(root, "<div class='hello world'>hello</div>");
 });
 
+test("attribute is removed if the binding becomes falsy", function() {
```

Comment:
👍 Also added an assertion to make sure there is coverage around non `class` attributes.